### ISSUE TITLE
Modulemd.ModuleStream.read_string does not accept None for optional arguments

### ISSUE THREAD

**Issue by mprahl** [fedora-modularity/libmodulemd#273]
`Modulemd.ModuleStream.read_string` has two optional arguments, `module_name` and `module_stream`. When interfacing with them in Python, `None` values are not allowed. Empty strings do work however. Here is the reproducer:

```python
import requests
import gi
gi.require_version("Modulemd", "2.0")
from gi.repository import Modulemd

mmd_str = requests.get(
    'https://pagure.io/fm-orchestrator/raw/master/f/tests/staged_data/platform.yaml').text
Modulemd.ModuleStream.read_string(mmd_str, None, None)
```

This causes the following traceback:
```
Traceback (most recent call last):
  File "reproducer.py", line 8, in <module>
    Modulemd.ModuleStream.read_string(mmd_str, None, None)
TypeError: Argument 1 does not allow None as a value
```

**Comment by sgallagher** [comment]
The problem here is that you are actually missing the `strict` argument, which is non-nullable. You probably want to do `Modulemd.ModuleStream.read_string(mmd_str, False, None, None)` here. (`Modulemd.ModuleStream.read_string(mmd_str, False)` will also work.

**Comment by sgallagher** [comment]
Actually, for MBS we probably want to have `True` there. We want `False` for DNF to be more tolerant, but on the packager side I think it makes sense to reject unknown values.

**Comment by mprahl** [comment]
Thanks @sgallagher. I fixed the documentation example in #276.
